File: JvmClient/src/commonMain/kotlin/spaceEngineers/model/JoinGameData.kt
package spaceEngineers.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable


@Serializable
data class JoinGameData(
    @SerialName("SelectedTab")
    val selectedTab: Int,
    @SerialName("SelectedTabName")
    val selectedTabName: String,
    @SerialName("Games")
    val games: List<ListedGameInformation>,
    @SerialName("JoinWorldButton")
    val joinWorldButton: GuiControlBase,
)

File: Source/Ivxr.SpaceEngineers/WorldModel/Screen/JoinGameData.cs
using System.Collections.Generic;

namespace Iv4xr.SpaceEngineers.WorldModel.Screen
{
    public class JoinGameData
    {
        public int SelectedTab;
        public string SelectedTabName;
        public List<ListedGameInformation> Games;
        public GuiControlBase ServerDetailsButton;
        public GuiControlBase DirectConnectButton;
        public GuiControlBase RefreshButton;
        public GuiControlBase JoinWorldButton;
    }
}
